### ISSUE TITLE
Support for sign-in links without user accounts

### ISSUE THREAD

**Issue by tjarksaul** [OpenSlides/OpenSlides#5385]
**Is your feature request related to a problem? Please describe.**
People often already have a lot of accounts. If they're just participating in one assembly in one year and maybe get replaced after that it isn't necessary for them to get another account for the assembly system. 

**Describe the solution you'd like**
I'd like the option to create users which do not have a "real" user name and especially not a password but just get a link via email with a token that logs them into the system for a specific time frame (Maybe configurable, maybe 24 hours before and after the session). 

**Describe alternatives you've considered**
Another solution would be a link with a token that is only valid once. If the user wants to log in again they would have to enter their email address and get a new token. But this would maybe too much overhead if the token is just time constrained as suggested above. 
I also thought about #4736 which also is a valid solution for that problem but not all organizations have a OpenID-compatible account system.

**Comment by FinnStutzenstein** [comment]
I think this would be a nice addition. I have a question about the authenticated users (with the token).

Quick terminology:
- participant: the real-world user sitting in front of his/her computer
- user: An OpenSlides-object representing an participant (in most cases)

OpenSlides has the "anonymous user", which is active, if you enable guests. So a participant opens OpenSlides and get a session assigned to the anonymous user. He/She cannot add himself on the list of speakers, because he/she is anonymous. To do this, you need a valid user in OpenSlides. This affects many interactive things in OpenSlides.

If a participant opens OpenSlides with the token (guests are not enabled) he is authenticated and can use OpenSlides, but which user does he/she become? Should the user be the anonymous one, so even the participant has a special token, can really interact read-only with OpenSlides, because the anonymous user is (often) not allowed to do anything (this depends on your permission configuration).

Or should a user be assigned to the token, so if a participant uses token X, he gets authenticated as  a user Y in OpenSlides? This means, that not only the token has to be generated, but also a user must be created (if they do not exist) and linked to the token.

**Comment by tjarksaul** [comment]
> Or should a user be assigned to the token, so if a participant uses token X, he gets authenticated as a user Y in OpenSlides? This means, that not only the token has to be generated, but also a user must be created (if they do not exist) and linked to the token.

Yes, this is basically my idea. I download the CSV from my participant registration and upload it to OpenSlides. OpenSlides then create users (without username/with random username and without password) with a token and sends a sign-in link with that token to the users so they can participate as authenticated, not anonymous users in the assembly.

**Comment by FinnStutzenstein** [comment]
Ok, I understand.

@emanuelschuetze Your opinion?

@GabrielInTheWorld Can you check, if this concept can be done with the OS4-auth service?

**Comment by Elblinator** [comment]
We now have a public access and we do not want these empty shells for users
